### DISCUSSION TITLE
System not converging / generator power increasing far too much.

### DISCUSSION THREAD

**Discussion by oh304** [idaholab/moose#28144] [Q&A Modules: Thermal Hydraulics] [answered]
Hi there,

I am simulating a helium primary loop, heated by a reactor core, which is then linked to a secondary loop almost entirely copied from the closed brayton example in MOOSE. Things seem to work OK so far, however, the generator power is increasing extortionately and I cannot understand for the love of god why! If anyone would like to help that would be greatly appreciated.

Below is the example I used to build the secondary loop.
https://mooseframework.inl.gov/moose/modules/thermal_hydraulics/examples/brayton_cycle/brayton_cycle.html

If you are interested in checking out the system, I have dropped it in this github repo below:
https://github.com/oh304/moose_sandbox

All the best,

Oscar

**Comment by GiudGiud** [comment]
@licharlot

**Comment by licharlot** [comment, marked as answer]
Hi Oscar, 
I think we exchanged a few emails. Hope the VTB example will help.
